basic/bank/tests/lib.rs

```diff
@@ -0,0 +1,34 @@
+use radix_engine::ledger::*;
```

Comment:
Can you remove this file? This is the default test file and doesn't work with your example

Reply:
done. Thanks for the hint.